.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Let's delete this file and other `.vs/*` files. I thought I added this to `.gitignore` already but maybe `.vs/` doesn't work the way I thought.